sdcm/cluster.py

```diff
@@ -2965,7 +2965,7 @@ def _node_setup(self, node, seed_address):
         node.remoter.send_files(src=yaml_dst_path,
                                 dst='/tmp/scylla.yaml')
         node.remoter.run('sudo mv /tmp/scylla.yaml /etc/scylla/scylla.yaml')
-        node.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic eth0 --disks `ls /dev/nvme0n* | tr "\n" ","`')
+        node.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic eth0 --disks `ls /dev/nvme0n* | tr "\n" "," | sed s/,$//`')
```

Comment:
It will be simple to generate the disks argument string by python.

```
dev_list = glob.glob("/dev/nvme0n*")
disks_str = ",".join(dev_list)
node.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic eth0 --disks %s' % disks_str)
```